strings_lab.js

```diff
@@ -0,0 +1,46 @@
+// # String Exercises
```

Comment:
I can see that you understand properties and methods for string manipulation but I also see instances of hard coding which is a practice to be avoided as a developer. Your code must return consistent and correct output despite different inputs. (i.e. code works whether you are inputting "JRJ" or "Briany"). Also try to do bonus questions as they are good practice. Keep coding, keep practicing and keep submitting assignments!